internal/postprocessor/go.mod

```diff
@@ -0,0 +1,5 @@
+module cloud.google.com/go/internal/postprocessor
+
+go 1.19
+
+// require cloud.google.com/go/internal/gapicgen v0.0.0-20221004191212-19692735b0fb
```

Comment:
nit: remove this line

Reply:
Also the go.sum can be removed for now if we don't have any deps we are pulling in.